.github/workflows/prepare-release.yaml

```diff
@@ -0,0 +1,84 @@
+name: prepare-release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version (e.g. 'v1.2.3-rc.4')"
+        required: true
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: weaveworks/wego-app
+
+jobs:
+  release-tasks:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Unshallow
```

Comment:
I think this can be replaced  by adding this to the `checkout` step
```yaml
with:
  fetch-depth: 0
```

Reply:
So. According to docs, yes. From past experience, maybe.

Last time I tried to do that I found that it still didn't fetch all the tags I expected, and the actions/checkout repo is full of issues saying the same, many closed without comment.

So, I'm going to leave this, because this seems to work.